docs/resources/epg_useg_criterion.md

```diff
@@ -0,0 +1,143 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "Application Management"
+layout: "aci"
+page_title: "ACI: aci_epg_useg_criterion"
+sidebar_current: "docs-aci-resource-aci_epg_useg_criterion"
+description: |-
+  Manages ACI EPG uSeg Criterion
+---
+
+# aci_epg_useg_criterion #
+
+Manages ACI EPG uSeg Criterion
```

Comment:
In ansible ( like in UI ) there is the concept of block statements ( crtrn and sub criterion ) and simple statements ( the attribute adds like ip/mc/dns etc..) Should we align the terminology with this?